docs/index.md

```diff
@@ -1,443 +1,20 @@
-        /* ... other dev plugins ... */
-      ]
-    },
-  }
-```
-If your project does not include a .babelrc file and you have a webpack.config.js file instead, you can add the `['@codesee/instrument', { hosted: true }]`, to this file. Look for an options block where your babel settings, presets and plugins are being declared. Please add `['@codesee/instrument', { hosted: true }]` to the list of your project's plugins as in the example below.
-
-```
-   options: {
-      presets: [
-        //your project's babel presets go here
-      ],
-      plugins: [
-        ['@codesee/instrument', { hosted: true }],
-        //other babel plugins go here
-      ],
-    },
-```
-
-## Did it work?
-Re-build your app and you should see the CodeSee eye icon towards the top right of your screen. Then you can celebrate!
-
-A few thoughts to help you on your CodeSee journey:
-
-- If the CodeSee icon is in a red circle then you're not logged in. Click it to log into CodeSee.
-- If the CodeSee icon is in a purple circle, you're ready to go! Click it to start recording, interact with your app, then click it again to stop recording.
-- Keep recordings as short as possible -- focus on the one thing you want to understand better.
-- When you finish recording, CodeSee will open a new tab with your recording in it as soon as all the data has been transmitted. For larger recordings or on slower networks, there can be some buffering, so please give it a few seconds or more to finish sending the recording to our servers.
-  - If it delays for long enough, your pop-up blocker may decide to block it. Keep an eye out for that if the codesee site isn't coming up. If in doubt, you can always go to [app.codesee.io/library](app.codesee.io/library) to see all your latest recordings.
-- The CodeSee UI can be a little sluggish with larger recordings! Sorry about that! For the moment, we ask that you give our UI a second to process now and then. But fear not, we take performance very seriously and we're actively working on performance improvements.
-
-## Compatibility / Troubleshooting
-### We're here to help!
-If there's anything you need, or you just want to tell us what you think, please don't hesitate to reach out to us at support@codesee.io
-
-### Browsers
-Currently, for CodeSee to work properly, you'll need to be running your application in Chrome.
-
-### Third-Party Cookies
-Unfortunately, if third-party cookies are blocked, our authentication process does not work.
-
-If you have disabled cross-site cookies, you'll need to add the following to the "allow these sites to always use cookies" list in Chrome settings to get CodeSee to work:
-
-```
-app.codesee.io
-[*.]okta.com
-```
-
+- If it delays for long enough, your pop-up blocker may decide to block it. Keep an eye out for that if the codesee site isn't coming up. If in doubt, you can always go to [app.codesee.io/library](https://app.codesee.io/library) to see all your latest recordings.
```

Comment:
This was meant to be a sub-bullet... but, if a top-level bullet, maybe just add another word or two for context:

"If the buffering takes long enough"

Reply:
I saw it was indented more in the markdown, but it wasn't being displayed as a sub-bullet. I can look into it.

Reply:
Ah it needs 4 spaces. Fixing